Library/Formula/xorg-util-macros.rb

```diff
@@ -0,0 +1,19 @@
+class XorgUtilMacros < Formula
+  desc "Miscellaneous utility macros required by the X.org codebase."
+  homepage "http://www.x.org"
+  url "http://xorg.freedesktop.org/archive/individual/util/util-macros-#{version}.tar.gz"
+  version "1.19.0"
+  sha256 "0d4df51b29023daf2f63aebf3ebc638ea88efedfd560ab5866741ab3f92acaa1"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "true"
```

Comment:
This is really only installing 3 files:

ln -s ../../Cellar/xorg-util-macros/1.19.0/share/aclocal/xorg-macros.m4 xorg-macros.m4
ln -s ../../Cellar/xorg-util-macros/1.19.0/share/pkgconfig/xorg-macros.pc xorg-macros.pc
ln -s ../Cellar/xorg-util-macros/1.19.0/share/util-macros util-macros

There is seriously nothing to test. I kept the test section just to make the audit pass.